contentcuration/contentcuration/frontend/channelEdit/views/TreeView/TreeViewBase.vue

```diff
@@ -491,6 +497,13 @@
       trackClickEvent(eventLabel) {
         this.$analytics.trackClick('channel_editor_toolbar', eventLabel);
       },
+      reloadChannel(id) {
+        if (!id) {
+          this.$store.dispatch('currentChannel/loadChannel').catch(error => {
```

Comment:
Maybe using `mapActions('currentChannel', [ 'loadChannel' ]),` and calling `this.loadChannel()` here might be a better way? A promise would still be returned so we should be good with the `catch`. I am only conflicted on throwing the error(next line) in this scenario, probably displaying a snackbar would be subtler(for example in situations when the server is disconnected)? @marcellamaki any thoughts?